core/src/main/java/tc/oc/pgm/command/util/CommandGraph.java

```diff
@@ -227,13 +226,8 @@ public void registerCommands() {
     register(new TeamCommand());
     register(new TimeLimitCommand());
     register(new VotingCommand());
+    register(new VanishCommand());
```

Comment:
Good point! So when third-party plugins set a vanish integration, how should that be handled? Ideally making it so the command is the responsibility of the plugin owning the vanish integration? Or do we always want to use the `VanishCommand` defined in PGM, and just have a method in the Integration to register/unregister it?

Reply:
It would be nice if we could make pgm self-register & self-unregister when a vanish manager is set, but that's kinda complicated when commands are involved.
I'd just keep it as part of the config, if you don't want conflicting commands between pgm's /v and your own plugins', just disable it in the config.

Reply:
Gotcha! That makes sense 👍